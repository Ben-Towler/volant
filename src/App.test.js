import React from 'react';
import App from './App';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('App', () => {

  describe('App Matches Snapshot', () => {
    test('snapshot matches', () => {
      const renderer = new ShallowRenderer();
      const result = renderer.render(<App />);
      expect(result).toMatchSnapshot();
    });
  });

});
