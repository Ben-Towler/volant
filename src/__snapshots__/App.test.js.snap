// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App Matches Snapshot snapshot matches 1`] = `
<div
  className="App"
/>
`;
